modules/scheduler/batch-job-template/variables.tf

```diff
@@ -91,10 +91,12 @@ variable "log_policy" {
   }
 }
 
-variable "runnable" {
```

Comment:
As written, this is a breaking change. I would suggest one of the following:
1. Breaking change with warning: If we are going to make a breaking change, then generally we would leave this variable, but put in the description that it is 'DEPRECATED' and add a validator that checks that input value is always null. This would allow us to provide an error message if the variable is used saying something like "such and such is deprecated, use such and such instead"
2. We could take both the plural and singular forms and enforce that only one is set. This would take a bit of tf code which I could help write.

I think if we end up making plural version more complicated (as I suggest in separate comment) then I would lean toward accepting both so the simple form stays simple.

Reply:
I agree, leaning towards option 2 assuming we will add additional configurations for runnables later, but some users will want the shorter/simpler form.

Reply:
SG. It looks like this still needs to be implemented. Right now runnable is still being deleted.